porting/first-steps.rst

```diff
@@ -70,6 +70,36 @@ Install the required dependencies::
      python-markdown libxml2-utils xsltproc zlib1g-dev:i386 schedtool \
      repo liblz4-tool bc lzop imagemagick libncurses5 rsync
 
+.. Note::
+    repo is not available in repositories of Ubuntu 20.04 or later versions. Hence it has to be installed manually.
+
+
+Installing repo 
+^^^^^^^^^^^^^^^^^
+
+To install repo, we need to ensure that you curl and git is installed. To install curl & git run the following command :
+
+.. code-block::
+
+    sudo apt install curl && sudo apt install git
```

Comment:
sudo apt install curl git

would be easier. Actually, you aren't using git though. Actually, actually, we just installed curl (and git) four lines above, so I think we could drop this whole paragraph

Reply:
Corrected as part of next commit.